app/views/media/_buttons.html.erb

```diff
@@ -0,0 +1,18 @@
+<span class="buttons">
+	<ul class="row small-up-1 medium-up-2">
+		<li class="column">
+			<%= link_to(image_tag("/assets/upvote_cupid_dart_heart.png"), image_path(upvote_path(movie.id)), action: :upvote, method: :post) %>
+		</li>
+		<li class="column">
+			<%= link_to(image_tag("/assets/downvote_broken_heart.png"), image_path(downvote_path(movie.id)), method: :post, action: :downvote) %>
+		</li>
+		<!-- edit button -->
+		<li class="column">
+			<%= button_to("Edit", edit_path(movie.id), method: :get, action: :edit) %>
```

Comment:
Same deal as above with `action: :upvote`, the `action: :edit` part on here is not doing anything for us here.